staging/src/k8s.io/apiserver/pkg/server/options/serving.go

```diff
@@ -75,6 +78,11 @@ type GeneratableKeyCert struct {
 	// CertDirectory is a directory that will contain the certificates.  If the cert and key aren't specifically set
 	// this will be used to derive a match with the "pair-name"
 	CertDirectory string
+	// FixtureDirectory is a directory that contains test fixture used to avoid regeneration of certs during tests.
+	// The format is:
+	// <host>_<ip>-<ip>_<alternateDNS>-<alternateDNS>.crt
+	// <host>_<ip>-<ip>_<alternateDNS>-<alternateDNS>.key
```

Comment:
should it be ```<host>_<ip>_<ip>_<alternateDNS>_<alternateDNS>```?

Reply:
fixed

Reply:
hrmm, just noticed this is adding test-only code into config structs... is this going to show up in external config types?

Reply:
These are not used externally in any config.